spring-graphql-test/src/main/java/org/springframework/graphql/test/tester/DefaultGraphQlTester.java

```diff
@@ -541,7 +541,7 @@ public EntityList<E> contains(E... values) {
 			doAssert(() -> {
 				List<E> expected = Arrays.asList(values);
 				AssertionErrors.assertTrue(
-						"Expecting list " + getEntity() + " at path '" + getPath() + "' to contain " + expected,
+						"Expecting list " + getEntity() + " at path '" + getPath() + "' to contain " + expected + ", but contains " + getEntity(),
```

Comment:
Isn't it printed already? `getEntity()` was in the message before the change, and is in it twice after the change. Or is it just a matter of improving the message? Looking at AssertJ, it uses this:

```
Expecting actual:
  [...]
to contain exactly (and in same order):
  [...]
but some elements were not expected:
  [...]
```

Reply:
Upon checking this it seems that there's already all the information in the message indeed. Could it be that this wasn't present in an earlier release, even tho the commit introducing it was tagged with the v1.0.x range?

Reply:
No worries, we can look into aligning messages with AssertJ, but that would be a separate issue.